crates/bevy_asset/src/io/embedded/mod.rs

```diff
@@ -168,6 +168,12 @@ pub fn _embedded_asset_path(
     file_path: &Path,
     asset_path: &Path,
 ) -> PathBuf {
+    let file_path = if cfg!(target_family = "wasm") {
+        // Work around bug: https://github.com/bevyengine/bevy/issues/14246
```

Comment:
Can we add an explicit comment here saying we know this will break any paths on Linux/Mac with "\\"? Definitely an edge case, but if someone hunts a panic here I want them to see this is something we considered.